hbase-server/src/main/java/org/apache/hadoop/hbase/master/normalizer/SimpleRegionNormalizer.java

```diff
@@ -592,5 +621,14 @@ public RegionStates getRegionStates() {
     public double getAverageRegionSizeMb() {
       return averageRegionSizeMb;
     }
+
+    public <T> T getOrDefault(String key, Function<String, T> function, T defaultValue) {
```

Comment:
nice.

Maybe this method returns an `Object`/`null` and the caller handles the null-check. I think this approach would be better than using an out-of-range value to indicate the lack of configuration.